### PR TITLE
Moving the CompressionFormat from package enum to file

### PR DESCRIPTION
**Pull request by giritherbhagi**

### CODE REVIEW

**Comment by realstraw** [comment]
Why?

**Comment by giritherbhagi** [comment]
@realstraw the package name cant be enum, java confuses with their own keyword. I was not able to import the package in the repository I was trying to use it

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 5.7.1 - 2019-11-08
+### Changed
+- [#590](https://github.com/krux/hyperion/issues/595) - Changing hyperion level enum to enumeration
```
Comment (realstraw) [inline CHANGELOG.md]: Let's make sure to have the right issue linked

File: contrib/activity/file/src/main/scala/com/krux/hyperion/contrib/activity/file/enumeration/CompressionFormat.scala [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.krux.hyperion.contrib.activity.file.enum
+package com.krux.hyperion.contrib.activity.file.enumeration
```
Comment (realstraw) [inline contrib/activity/file/src/main/scala/com/krux/hyperion/contrib/activity/file/enumeration/CompressionFormat.scala]: let's just put this in `com.krux.hyperion.contrib.activity.file`

**Comment by giritherbhagi** [comment]
@realstraw  made the changes you suggested

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,9 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 5.7.2 - 2019-11-08
+### Changed
+- [#590](https://github.com/krux/hyperion/issues/595) - Not able to import compressionFormat enum in other repository which uses hyperion
```
Comment (realstraw) [inline CHANGELOG.md]: should be `#595`

**Pull request merged**